mokapot/brew.py

```diff
@@ -114,35 +114,37 @@ def brew(psms, model=None, test_fdr=0.01, folds=3, max_workers=1):
 
         raise err from orig_err
     except TypeError:
-        models = Parallel(n_jobs=max_workers, require="sharedmem")(
+        fitted = Parallel(n_jobs=max_workers, require="sharedmem")(
             delayed(_fit_model)(d, copy.deepcopy(model), f)
             for f, d in enumerate(train_sets)
         )
 
     # sort models to have deterministic results with multithreading.
```

Comment:
Maybe capitalize "sort" here to make it consistent with the rest of the comments in the project?